ssl_cert_tracker/db_helper.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+from datetime import datetime
+import time
+import logging
+
+from django.db.models.signals import post_save
+
+from django.core.signals import request_finished
+from django.dispatch import receiver
+
+from django.core.exceptions import ObjectDoesNotExist
+from ssl_cert_tracker.models import TestCertsData, TestCertsHistory
+
+logging.basicConfig(filename='pocProject.log',level=logging.DEBUG)
+
+def Insert_Into_CertsData(json_data):
+    """Insert_Into_CertsData """
+    return_code = "Success"
+    try:
+        db_certs = TestCertsData()
+        db_certs.orion_id = json_data["orion_id"]
+        db_certs.addresses = json_data["addresses"]
+        
+        if "not_before" in json_data and validate(json_data["not_before"]) == True:

```

Comment:
please explain this code. whoever will have to maintain will have no idea why you are doing it this way.

oh yeah, you are using it more than once; please isolate it into its on function

Reply:
Very simple, I already told you, 
Getting date fields  values from xml data are not guarantee to be in date format 
so I check them first to make sure they are valid then get its posix value and save them into the model
Why do I need to get the posix value first... a good question...Django does not let me do it otherwise unless I use other util module, since this method works then I am fine with it